content/en/docs/_index.md

```diff
@@ -12,6 +12,12 @@ menu:
 
 Read more in the [introduction blog post](https://operatorhub.io/what-is-an-operator).
 
+## When to use OLM
+
+
+The premise of an Operator is to have it be a custom form of Controllers, a core concept of Kubernetes. A controller is basically a software loop that runs continuously on the Kubernetes master nodes. In these loops the control logic looks at certain Kubernetes objects of interest. It audits the desired state of these objects, expressed by the user, compares that to what’s currently going on in the cluster and then does anything in its power to reach the desired state.
+This declarative model is basically the way a user interacts with Kubernetes. Operator solves this problem and basically replaces a "human" operator responsible to perform certain task like provisioning and configuring with a "software" operator.
```

Comment:
This doesn't really explain to me "when to use OLM", it simply states what an Operator is.

Reply:
Thank you for the comment. I am working on improving the article.